app/http.php

```diff
@@ -54,8 +54,26 @@
     $app = new App('UTC');
 
     go(function() use ($register, $app) {
-        $db = $register->get('dbPool')->get();
-        $redis = $register->get('redisPool')->get();
+        // wait for database to be ready
+        $attempts = 0;
+        $max = 10;
+        $sleep = 1;
+
+        do {
+            try {
+                $attempts++;
+                $db = $register->get('dbPool')->get();
+                $redis = $register->get('redisPool')->get();
+                break; // leave the do-while if successful
+            } catch(\Exception $e) {
+                Console::warning("Database not ready. Retrying connection ({$attempts})...");
+                if ($attempts >= $max) {
+                    throw new \Exception('Failed to connect to database: '. $e->getMessage());
+                }
+                sleep($sleep);
+                continue;
```

Comment:
I think the continue here is not necessary :thinking:

Reply:
Ah, t'was left over from a previous attempt at logic structure. Removed in 4242ba7